### PR TITLE
Features #5 #6 #7 #8 & Updates

### PR DESCRIPTION
**Pull request by tdigangi**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -60,3 +52,8 @@ variable "table_labels" {
   description = "Key value pairs in a map for table labels"
   type        = "map"
 }
+
+variable "tables" {
+  description = "A list of table IDs that will be created on the single dataset"
```
Comment (aaron-lane) [inline variables.tf]: Based on the usage of this variable in `main.tf`, this is actually a list of maps which include a `table_id` and a `schema`. This expected structure needs to be documented in the description.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
...
+- Support for multiple tables per [Issue #6](https://github.com/terraform-google-modules/terraform-google-bigquery/issues/6)
+- Examples to support google provider >2.5.0 per [Issue #8](https://github.com/terraform-google-modules/terraform-google-bigquery/issues/8)
+- Upgraded Gemfile/Gemfile.lock for kitchen-terraform gems
+- Module output for table name
+
+### Changed
+- Module ONLY accepts a _list of maps_ for the table_name, table_id, and schema. This enables the creation of multiple tables on a single dataset.
+- Inspec attributes for testing in inspec.yml
+
+### Improved
```
[the remark this answers is not in the document]
Comment (tdigangi) [reply CHANGELOG.md]: will be changed in next commit

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.0] 2019-04-30
+
+### Added
+- Support for multiple tables per [Issue #6](https://github.com/terraform-google-modules/terraform-google-bigquery/issues/6)
+- Examples to support google provider >2.5.0 per [Issue #8](https://github.com/terraform-google-modules/terraform-google-bigquery/issues/8)
+- Upgraded Gemfile/Gemfile.lock for kitchen-terraform gems
```
[the remark this answers is not in the document]
Comment (tdigangi) [reply CHANGELOG.md]: will be changed in next commit

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
...
+- Module output for table name
+
+### Changed
+- Module ONLY accepts a _list of maps_ for the table_name, table_id, and schema. This enables the creation of multiple tables on a single dataset.
+- Inspec attributes for testing in inspec.yml
+
+### Improved
+- Examples for multiple tables pre [Issue #5](https://github.com/terraform-google-modules/terraform-google-bigquery/issues/5)
```
[the remark this answers is not in the document]
Comment (tdigangi) [reply CHANGELOG.md]: will be changed in next commit

File: README.md [hunk 5]
```diff
@@ -49,15 +51,18 @@ A helper script for configuring a Service Account is located at (./helpers/setup
...
-Be sure you have the correct Terraform version (0.11.x), you can choose the binary from [Terraform releases](https://releases.hashicorp.com/terraform/).
+Be sure you have the current Terraform version (0.11.x), you can choose the binary from [Terraform releases](https://releases.hashicorp.com/terraform/).
 
 ### kitchen-terraform
 To set this up on your machine, follow the official [Kitchen installation](https://github.com/newcontext-oss/kitchen-terraform) instructions.
 - Kitchen tests are located: [test/integration/full](test/integration/full).
 - Terraform fixtures are located: [test/fixtures/full](test/fixtures/full).
 
-## Running tests
+### macOS mojave notes
```
[the remark this answers is not in the document]
Comment (tdigangi) [reply README.md]: will be changed in next commit

File: examples/multiple_tables/README.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# terraform-google-bigquery multiple_tables
+The multiple_tables example uses the root terraform-google-bigquery module to deploy a single dataset and a table with a basic schema. This example is a good reference to understand and test the module usage.
```
[the remark this answers is not in the document]
Comment (tdigangi) [reply examples/multiple_tables/README.md]: will be changed in next commit

File: variables.tf (hunk 1 above)
Comment (tdigangi) [reply variables.tf]: will be changed in next commit

**Comment by tdigangi** [comment]
Added upgrade guide and have made requested changes. Ready for next review @aaron-lane

**Comment by aaron-lane** [comment]
Fixes #5 
Fixes #6 
Fixes #7 
Fixes #8

File: docs/upgrading_to_bigquery_v1.0.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+
+```hcl
+/// @file main.tf
+
+provider "google" {
+  version = "~> 1.20.0"
+}
+
+module "bigquery" {
+  source            = "github.com/terraform-google-modules/terraform-google-bigquery.git?ref=v0.1.0"
```
[the remark this answers is not in the document]
Comment (tdigangi) [reply docs/upgrading_to_bigquery_v1.0.md]: done

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
...
+- Module ONLY accepts a _list of maps_ for the table_name, table_id, and schema. This enables the creation of multiple tables on a single dataset.
+- Inspec attributes for testing in inspec.yml
+- Examples for multiple tables per [Issue #5](https://github.com/terraform-google-modules/terraform-google-bigquery/issues/5)
+- Testing with native inspec resources per [Issue #7](https://github.com/terraform-google-modules/terraform-google-bigquery/issues/7)
+- Support for multiple tables per [Issue #6](https://github.com/terraform-google-modules/terraform-google-bigquery/issues/6)
+- Examples to support google provider >2.5.0 per [Issue #8](https://github.com/terraform-google-modules/terraform-google-bigquery/issues/8)
+- Upgraded Gemfile/Gemfile.lock for kitchen-terraform gems
+- Module output for table name
+
+[1.0.0]: https://github.com/terraform-google-modules/terraform-google-bigquery/tree/v0.2.0/
```
[the remark this answers is not in the document]
Comment (tdigangi) [reply CHANGELOG.md]: done

**Comment by tdigangi** [comment]
@aaron-lane committed all changes requested

**Pull request merged**